src/wad/image.rs

```diff
@@ -125,15 +151,23 @@ impl Image {
             height: height,
             x_offset: x_offset,
             y_offset: y_offset,
-            pixels: pixels
+            pixels: pixels,
         })
     }
 
     pub fn blit(&mut self, source: &Image, offset: Vec2<isize>, ignore_transparency: bool) {
         // Figure out the region in source which is not out of bounds when
         // copied into self.
-        let y_start = if offset[1] < 0 { (-offset[1]) as usize } else { 0 };
-        let x_start = if offset[0] < 0 { (-offset[0]) as usize } else { 0 };
+        let y_start = if offset[1] < 0 {
```

Comment:
Could you please fix these up to the way they were? I think rustfmt definitely needs a "ternary-like if statement detector".